app/Repositories/Eloquents/WebhookRepository.php

```diff
@@ -20,4 +20,8 @@ public function getAllByUser()
                             ->orderBy('webhooks.created_at', 'desc')
                             ->get();
     }
+
+    public function getWebhookByName($name) {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/WebhookRepository.php</br>**Line**: 24</br>Opening brace should be on a new line